Ingress/controllers/nginx-third-party/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2015 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/google_containers/nginx-slim:0.1
+
+COPY nginx-third-party-lb /
+COPY nginx.tmpl /
+COPY default.conf /etc/nginx/nginx.conf
+
+WORKDIR /
+
+EXPOSE 80 443 11249
```

Comment:
why expose here? you're forcing everyone to do things through these ports. Suggest surfacing it through yaml instead.